lib/package.json

```diff
@@ -50,7 +50,7 @@
     "test:ci": "jest --coverage && codecov",
     "test:e2e": "jest --config jest.config.e2e.js --runInBand --detectOpenHandles --forceExit",
     "doc": "./node_modules/.bin/esdoc",
-    "bootstrap": "lerna bootstrap && lerna exec npm install",
+    "bootstrap": "lerna bootstrap && lerna exec npm install && lerna exec tsc",
```

Comment:
I think `lerna run tsc` will work better because it doesn't require tsc to be installed as a global or added to the $PATH